src/ui/pages/SendForm/shared/useAddressPositionsFromBackendOrNode.ts

```diff
@@ -16,28 +17,36 @@ export function useAddressPositionsFromBackendOrNode({
   chain: Chain | null;
 }): { data: AddressPosition[] | null | undefined; isLoading: boolean } {
   const { networks } = useNetworks();
+  const hasChain = Boolean(chain);
   const isSupportedByBackend =
     chain && networks ? networks.supports('positions', chain) : null;
   const { data, isLoading } = useHttpAddressPositions(
     { addresses: [address], currency },
     { source: useHttpClientSource() },
     {
       // we query positions for all chains, so we can do it even before the "supported" check is ready
-      enabled: isSupportedByBackend == null || isSupportedByBackend,
+      enabled: hasChain && isSupportedByBackend === true,
```

Comment:
Do we need `=== true` here?

Reply:
The value is nullish and expected one is boolean?